src/test/java/de/undercouch/actson/PrettyPrinter.java

```diff
@@ -23,12 +23,11 @@
 
 package de.undercouch.actson;
 
-import java.nio.charset.StandardCharsets;
```

Comment:
Please do not change this class too much. It's an example and the README links to it. It should be self-contained and have no dependencies (to Jackson).